pkg/csi/manila/controllerserver.go

```diff
@@ -43,6 +43,15 @@ type controllerServer struct {
 var (
 	pendingVolumes   = sync.Map{}
 	pendingSnapshots = sync.Map{}
+
+	// Recognized volume parameters passed by Kubernetes csi-provisioner sidecar
+	// when run with --extra-create-metadata flag. These are added to metadata
+	// of newly created shares if present.
+	recognizedCSIProvisionerParams = []string{
```

Comment:
looks like cinder CSI currently always honor the param now ...

Reply:
@jichenjc thanks for checking this! I wanted to retain the previous behavior -- no PV/PVC info in volume metadata. But I guess enabling it by default shouldn't cause issues. @gouthampacha do you think this is ok?